### PR TITLE
OCPBUGS-1130: increase etcdGRPCRequestsSlow thresholds

### PR DESCRIPTION
**Pull request by tjungblu**
for the update in https://github.com/openshift/cluster-etcd-operator/pull/932

### CODE REVIEW

File: alerts/cluster-etcd-operator/etcdGRPCRequestsSlow.md [hunk 1]
```diff
@@ -73,24 +85,47 @@ To confirm this is the cause of the slow requests either:
...
+
+Find the offending process that causes the disk performance to degrade,
+this can also be a noisy neighbour process on the control plane node
+(eg fluentd with logging, OVN) or etcd itself. If the culprit is determined
+to etcd, try to reduce the load coming from the apiserver.
+Most commonly this also happens when a cluster is scaled up with
+many more nodes, so reducing the cluster scale again can help.
+
+If feasible on clouds, upgrading your storage or instance type
+can significantly increase your sequential IOPS and available bandwidth.
```
Comment (wking) [inline alerts/cluster-etcd-operator/etcdGRPCRequestsSlow.md:122]: The process for scaling up a control plane is still lacking official docs, isn't it?

Comment (tjungblu) [reply alerts/cluster-etcd-operator/etcdGRPCRequestsSlow.md:122]: really? this is the first google hit: https://access.redhat.com/solutions/5377471

**Comment by tjungblu** [comment]
/assign @Elbehery 

if you have some time to spare

**Pull request merged**